modules/gdscript/README.md

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
Is this empty line accidental?

Reply:
Nope, really wasn't! :)

Reply:
You mean we need an empty line for some reason?

Reply:
Oh my goodness, my bad! I meant it really wasn't _intentional_, not _accidental_ 😅